### ISSUE TITLE
REST API: uploading a new file (upload) is not working in docker container

### ISSUE THREAD

**Issue by tngraf** [fossology/fossology#2187]
### Description

Uploading a new file (upload) is **not working** in the docker container, Commit: [#fcf1de] 2022/03/09 08:26.
It **is working** with a fully installed instance like the one running at Siemens for example.

#### How to reproduce

I tried as well the .Net method `UploadPackage` as also the Python library.

The result is always
```
{"code":500,"message":"No file selected.\n","type":"ERROR"}
```

### Versions

* Last commit id on master: Commit: [#fcf1de] 2022/03/09 08:26.
  (Docker container fossology/fossology:latest as of 2022-03-12
* Operating System (for the client): Windows 10

**Comment by GMishx** [comment]
Hello @tngraf , this issue was fixed by #2179 and is in the commit right after fcf1de.
Can you pull the image from 2022-03-14 or later?

**Comment by tngraf** [comment]
It is working with the latest image - closing the issue.
